### PR TITLE
Make a rpc url parameter for trading report

### PR DESCRIPTION
**Pull request by brittcyr**

### CODE REVIEW

File: trading_report/src/main.ts [hunk 1]
```diff
@@ -12,7 +12,7 @@ async function main() {
   console.log('Analysis running', new Date().toUTCString(), SYMBOL);
 
   const connection = new Connection(
-    'https://floral-skilled-borough.solana-mainnet.discover.quiknode.pro/38cf24edefbebeb60eb7516eff40f076ac0823af/',
```
Comment (DonDuala) [inline trading_report/src/main.ts:15]: Think we need to purge this here too: https://github.com/Dual-Finance/status/blob/6a4738f878e2992ee38b4961c9e52b334fa1bdd4/app/src/config/config.ts#L10

Comment (brittcyr) [reply trading_report/src/main.ts:15]: It is a client side fetch. Cant hide it

This change isnt about hiding it, it is about letting us use a parameter for the overnight runs when we run out of quota in a couple days

Comment (DonDuala) [reply trading_report/src/main.ts:15]: Ok are you worried about ppl scraping for private RPCs elsewhere?

Comment (brittcyr) [reply trading_report/src/main.ts:15]: Cant avoid it easily, so nothing to really worry about. But should keep it all separate so we have failure isolation.

Comment (DonDuala) [reply trading_report/src/main.ts:15]: Isolated by using a separate RPC for status/landing/app?

Comment (brittcyr) [reply trading_report/src/main.ts:15]: Correct. Each type of caller shouldnt be able to cause failures on another due to resource exhaustion when possible

**Pull request merged**